src/dss/main.py

```diff
@@ -30,5 +31,48 @@ def initialize_command(kubeconfig: str) -> None:
     initialize(lightkube_client=lightkube_client)
 
 
+CREATE_NOTEBOOK_IMAGE_HELP = """
+\b\nThe image used for the notebook server.  Suggested images:
+- kubeflownotebookswg/jupyter-scipy:v1.8.0 (default)
+- kubeflownotebookswg/jupyter-pytorch-full:v1.8.0
+- kubeflownotebookswg/jupyter-pytorch-cuda-full:v1.8.0
+- kubeflownotebookswg/jupyter-tensorflow-full:v1.8.0
+- kubeflownotebookswg/jupyter-tensorflow-cuda-full:v1.8.0
+"""
+
+
+@main.command(name="create-notebook")
+@click.argument(
+    "name",
+    required=True,
+)
+@click.option(
+    "--image",
+    default="kubeflownotebookswg/jupyter-scipy:v1.8.0",
+    help=CREATE_NOTEBOOK_IMAGE_HELP,
```

Comment:
The help message should be as close as possible to the ux spec doc. Please change the summary plus add example section (I know click will always put it before options :)).

Reply:
I added the example section, here's how it looks now:
```
dss create --help
Usage: dss create [OPTIONS] NAME

  Create a Jupyter notebook in DSS and connect it to MLflow. This command also
  outputs the URL to access the notebook on success.

  Examples:
      dss create my-notebook --image=pytorch
      dss create my-notebook --image=kubeflownotebookswg/jupyter-scipy:v1.8.0

Options:
  --image TEXT       The image used for the notebook server.
                     Recommended images:
                       - pytorch = charmedkubeflow/jupyter-pytorch-full:1.8.0-3058193
  --kubeconfig TEXT  Path to a Kubernetes config file. Defaults to the value
                     of the KUBECONFIG environment variable, else to
                     './kubeconfig'.
  --help             Show this message and exit.
```

Reply:
Modified this a bit 

```diff
-                       - pytorch = charmedkubeflow/jupyter-pytorch-full:1.8.0-3058193
+                       - pytorch: charmedkubeflow/jupyter-pytorch-full:1.8.0-3058193

```